var/spack/repos/builtin/packages/caffe/package.py

```diff
@@ -0,0 +1,79 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+class Caffe(CMakePackage):
+    """Caffe is a deep learning framework made with expression, speed, and
+       modularity in mind. It is developed by the Berkeley Vision and Learning
+       Center (BVLC) and by community contributors."""
+
+    homepage = "http://caffe.berkeleyvision.org"
+    url      = "https://github.com/BVLC/caffe/archive/rc5.tar.gz"
+
+    version('rc5', '692bd3580b7576485cde6b1e03eb5a6d')
+
+    variant('gpu', default=False, 
+            description='Builds with support for GPUs via CUDA and cuDNN')
+    variant('opencv', default=True, 
+            description='Build with OpenCV support')
+    variant('leveldb', default=True, 
+            description="Build with levelDB")
+    variant('lmdb', default=True, 
+            description="Build with lmdb")
+    variant('python', default=False, 
+            description='Build python wrapper and caffe python layer')
+    variant('matlab', default=False, 
+            description='Build Matlab wrapper')
+
+    depends_on('boost')
+    depends_on('boost +python', when='+python')
```

Comment:
Can u do: 'spac spec caffe' and 'space spec caffe+python'? In similar circumstances the concretizer did not work for me.